nerdlets/github-about/repo-picker.js

```diff
@@ -108,7 +109,7 @@ export default class RepoPicker extends React.PureComponent {
       buttonTitle = 'Clear';
     }
 
-    if (isUrlSafe(setRepoValue)) {
+    if (isUrlSafe(setRepoValue) || setRepoValue === '') {
```

Comment:
I _think_ that `isUrlSafe('')` will return `false` because `new URL('')` is not valid and [should throw](https://github.com/newrelic/nr1-github/blob/main/nerdlets/shared/utils.js#L8).

```suggestion
    if (isUrlSafe(setRepoValue)) {
```

Reply:
This was due to how the `setRepoValue` was being set to `''` when the repo was selected, without the `||` check it wasn't rendering at all so you couldn't clear a repo when it had been set. It probably makes more sense to just change that behaviour though